fhir-server/liberty-config/config/default/fhir-server-config.json

```diff
@@ -145,7 +145,8 @@
             "isExportPublic": true,
             "cosFileMaxResources": 500000,
             "cosFileMaxSize": 209715200,
-            "validBaseUrls": []
+            "validBaseUrls": [],
+            "useFhirServerTrustStore": false
```

Comment:
I think I'd prefer to make the default true.

Reply:
same